cluster-scope/overlays/prod/moc/smaug/externalsecrets/service-catalog-k8s-plugin-tokens.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: service-catalog-k8s-plugin-tokens
+  namespace: service-catalog
+spec:
+  secretStoreRef:
+    name: opf-vault-store
+    kind: SecretStore
```

Comment:
This secret store doesn't exist, please follow: https://www.operate-first.cloud/apps/content/vault_eso/add_external_secrets_to_ns.html#enable-namespace-to-fetch-secrets-from-vault

Reply:
Maybe I am missing something obvious but, doesn't the SecretStore exist here: https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/prod/moc/smaug/secret-mgmt/base/opf-vault-store.yaml?

Reply:
I believe what you are missing is [this](https://vault-ui-vault.apps.smaug.na.operate-first.cloud/ui/vault/secrets/k8s_secrets/list/moc/smaug/). If you look in that link I shared for the Vault UI, you will see no namespace  `service-catalog`. As stated in the documentation that Tom shared, you need to create a folder [here](https://github.com/operate-first/apps/tree/master/cluster-scope/overlays/prod/moc/smaug/secret-mgmt) for the `service-catalog` namespace, and a kusotmization file that references [the base folder](https://github.com/operate-first/apps/tree/master/cluster-scope/overlays/prod/moc/smaug/secret-mgmt/base).

Reply:
Oh yea, totally forgot about that part, thanks.

Reply:
In addition to what @Gregory-Pereira wrote, you need to (as described in the documentation linked above) update the policy in Vault with the namespace:

https://vault-ui-vault.apps.smaug.na.operate-first.cloud/ui/vault/access/smaug-k8s/item/role/edit/moc-ops

![image](https://user-images.githubusercontent.com/7453394/188109367-43ebb286-4ce3-435c-90c1-05f6b7baaa96.png)

See the docs:
https://www.operate-first.cloud/apps/content/vault_eso/add_external_secrets_to_ns.html#enable-namespace-to-fetch-secrets-from-vault
![image](https://user-images.githubusercontent.com/7453394/188109478-63aec47c-d516-4be5-b575-f17e58194de1.png)

Reply:
Namespace added in vault.